tests/endpoints/bookstore_grpc/proto/v1/bookstore.proto

```diff
@@ -36,6 +36,15 @@ service Bookstore {
     //   curl http://DOMAIN_NAME/v1/shelves
     option (google.api.http) = {
       get: "/v1/shelves"
+      // This is added to test grpc-web, to make sure the method is enabled
+      // for transcoder filter, and it should bypass this method.
+      // grpc-web client is using POST + /service_name/method_name.
+      // It could be auto-generated by setting auto_mapping. But auto_mapping
+      // will not apply if a method already has http.rules options defined.
+      additional_bindings {
```

Comment:
Is there any integration test using this rule, and can capture the grpc-web failure?

Reply:
Yes,  test/grpc_integration_test.go  its GRPCWeb test is using all these 3 additional_binding rules.  They are sending POST + /service_name/service_method HTTP request.  and pass to grpc transcoder.